backend/tests/integration/search/test_aggregations.py

```diff
@@ -434,6 +437,13 @@ def test_jobs_field_of_interest_aggregation_and_filter(inspire_app, override_con
             response["hits"]["hits"][0]["metadata"]["control_number"]
             == expected_record["control_number"]
         )
+        with inspire_app.test_client() as client:
+            response = client.get("/jobs?field_of_interest=Other").json
```

Comment:
do we have a test case with the Unknown?

Reply:
It's an old functionality which is working, and there's already a test for it https://github.com/inspirehep/inspirehep/blob/0be447100b82bfc965f46e0836a14dc8fa903959/backend/tests/integration/search/test_aggregations.py#L354-L355